lib/scenic/view.rb

```diff
@@ -14,11 +14,11 @@ def ==(other)
     end
 
     def to_schema
-      <<-DEFINITION.strip_heredoc
-        create_view :#{name}, sql_definition:<<-\SQL
-          #{definition}
-        SQL
+      <<-DEFINITION
 
+  create_view :#{name}, sql_definition: <<-\SQL
+    #{definition.indent(2)}
```

Comment:
This line needs to be dedented to 0 column for the `indent` to have the desired effect.